src/Tree/HydroTree.cpp

```diff
@@ -1153,7 +897,7 @@ void HydroTree<ndim,ParticleType,TreeCell>::UpdateHydroExportList
           {
             index_cell = Ncellexport[j]++;
           }
-          cellexportlist[j][index_cell] = cellptr;
+          cellexportlist[j][index_cell] = cell.id ;
```

Comment:
Is id simply the position in the array? This is a generic question rather than about this specific code, but I am just wondering, why do we need to store it?